### PR TITLE
Fix/refactoring files structure

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/testsession/models.py [hunk 1]
```diff
@@ -99,6 +99,12 @@ class VNGEndpoint(models.Model):
     session_type = models.ForeignKey(SessionType, on_delete=models.CASCADE)
     test_file = FilerFileField(null=True, blank=True, default=None, on_delete=models.SET_NULL)
 
+    def save(self, *args, **kwargs):
```
Comment (alextreme) [inline src/vng/testsession/models.py]: afaik this can be removed

File: src/vng/testsession/api_views.py [hunk 2]
```diff
@@ -421,9 +421,10 @@ def build_url(self, eu, arguments):
         ru = self.kwargs['relative_url']
         if eu.vng_endpoint.url is not None:
             if eu.vng_endpoint.url.endswith('/'):
-                request_url = '{}{}?{}'.format(eu.vng_endpoint.url, self.kwargs['relative_url'], arguments)
-            else:
-                request_url = '{}/{}?{}'.format(eu.vng_endpoint.url, self.kwargs['relative_url'], arguments)
+                eu.vng_endpoint.url = eu.vng_endpoint.url[:-1]
```
Comment (alextreme) [inline src/vng/testsession/api_views.py]: removing the slash at the end of the url can be removed

**Comment by alextreme** [comment]
As requested taking a good look at how the url and path fields should be entered is a good idea (and avoid the duplicate slashes when clicking through).

Also provide concrete examples in the help_text to avoid any misunderstandings.

**Comment by alextreme** [comment]
http://jenkins.maykin.nl/job/VNG%20Github/job/api-testvoorziening/view/change-requests/job/PR-145/

**Comment by AlviseSembenico** [comment]
@alextreme tests fixed

**Pull request merged**